automated-comments/go/two_fer/missing_function_comment.md

```diff
@@ -0,0 +1 @@
+Community style says that a function in Go should have a comment. For more details, have a look at the comments section below.
```

Comment:
> It's idiomatic in Go to provide a comment for each exported function. For more details, see the comments section below.

Question: do we only post the comment section if we pass back a comment about comments?

Reply:
Yes, that is currently the case.